extensions/standard-processors/processors/PutUDP.cpp

```diff
@@ -107,48 +98,48 @@ void PutUDP::onTrigger(core::ProcessContext* context, core::ProcessSession* cons
     return;
   }
 
-  const auto nonthrowing_sockaddr_ntop = [](const sockaddr* const sa) -> std::string {
-    return utils::try_expression([sa] { return utils::net::sockaddr_ntop(sa); }).value_or("(n/a)");
+  asio::io_context io_context;
+
+  const auto resolve_hostname = [&io_context, &hostname, &port]() -> nonstd::expected<udp::resolver::results_type, std::error_code> {
+    udp::resolver resolver(io_context);
+    std::error_code error_code;
+    auto resolved_query = resolver.resolve(udp::v4(), hostname, port, error_code);
+    if (error_code)
+      return nonstd::make_unexpected(error_code);
+    return resolved_query;
+  };
+
+  const auto debug_log_resolved_endpoint = [&hostname, &logger = this->logger_](const udp::resolver::results_type& resolved_query) -> udp::endpoint {
+    if (logger->should_log(core::logging::LOG_LEVEL::debug))
+      core::logging::LOG_WARN(logger) << "resolved " << hostname << " to: " << resolved_query->endpoint();
```

Comment:
should this be `LOG_DEBUG`?

Reply:
you are right, fixed it https://github.com/apache/nifi-minifi-cpp/pull/1412/commits/45948030831865d2a71d6a47135a2f6deac823d0#diff-e126536bed52648bd8b876e5a90622a3ebfc00207b7c0b10ca7f1f9605e67c0eR114